### PR TITLE
updating DGAB to 2015

### PR DESCRIPTION
**Pull request by spf13**
Signed-off-by: spf13 <steve.francia@gmail.com>

### CODE REVIEW

File: docs/governance/board-profiles.md [hunk 1]
```diff
@@ -38,34 +38,35 @@ governed properly working in concert with the Docker Governance Advisory Board
...
-## Contributor
+## Individual Contributors
 
 * Andrew Tianon Page
-* Brandon Phillips
-* Jonathan Rudenberg
-* David Calavera
+* Aleksa Sarai
+* Harald Albers
+* Darren Shepherd
```
Comment (thaJeztah) [inline docs/governance/board-profiles.md]: Darren is not representing Rancher?

Comment (spf13) [inline docs/governance/board-profiles.md]: No, he's on as an individual contributor.

**Comment by thaJeztah** [comment]
Would there be interest in a "former members" section? (Don't know if that's information people are interested in, just thinking out loud)

LGTM, one question that might need clarification added inline.

**Comment by thaJeztah** [comment]
ping @moxiegirl

**Comment by moxiegirl** [comment]
LGTM

**Pull request merged**